backend/tests/integration/stripe.integration.test.js

```
@@ -0,0 +1,364 @@
+      expect(mockStripe.checkout.sessions.create).toHaveBeenCalledWith(
+        expect.objectContaining({
+          success_url: 'https://test.example.com/success?session_id={CHECKOUT_SESSION_ID}',
+          cancel_url: 'https://test.example.com/cancel',
+        }),
+        expect.anything()
+      );
+    });
+
+    it('falls back to CLIENT_URL when specific URLs not set', async () => {
+      // Test when specific stripe URLs are not set
+      delete process.env.STRIPE_SUCCESS_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)